### PR TITLE
Add gke-gpu tests for 1.7.

### PR DESCRIPTION
**Pull request by mindprince**
This is related to https://github.com/kubernetes/kubernetes/issues/47388 and https://github.com/kubernetes/test-infra/pull/3059

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-gke-gpu-1-7.env [hunk 1]
```diff
@@ -0,0 +1 @@
+PROJECT=k8s-jkns-e2e-gke-gpus-1-7
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-gpu-1-7.env]: now we uses `--gce-project=k8s-jkns-e2e-gke-gpus-1-7` in config.json (ref https://github.com/kubernetes/test-infra/pull/3528)

File: jobs/config.json [hunk 2]
```diff
@@ -5379,6 +5379,21 @@
...
+      "--env-file=jobs/ci-kubernetes-e2e-gke-gpu.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gke-gpu-1-7.env",
+      "--timeout=180m",
+      "--extract=ci/latest-1.7",
+      "--check-leaked-resources",
+      "--test_args=--ginkgo.focus=\\[Feature:GPU\\]"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-node"
```
Comment (krzyzacy) [inline jobs/config.json]: I assume it's not sig-node?

Comment (mindprince) [inline jobs/config.json]: Updated.

File: jobs/ci-kubernetes-e2e-gke-gpu-1-7.env (hunk 1 above)
Comment (mindprince) [inline jobs/ci-kubernetes-e2e-gke-gpu-1-7.env]: Updated.

**Comment by krzyzacy** [comment]
ah, @mindprince you can remove that unit test as `job.env` is not required anymore.

**Pull request merged**